app/views/layouts/_head.html.erb

```diff
@@ -0,0 +1,17 @@
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <%#
+    Configure default and controller-, and view-specific titles in
+    config/locales/en.yml. For more see:
+    https://github.com/calebthompson/title#usage
+  %>
+  <title><%= title %></title>
+  <%= stylesheet_link_tag "application", "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= csrf_meta_tags %>
+  <link href="//fonts.googleapis.com/css?family=Roboto:400,700,300,400italic,700italic,300italic" rel="stylesheet" type="text/css">
```

Comment:
I believe you can combine these google fonts into one line using `|`'s
`<link href="//fonts.googleapis.com/css?family=Roboto:400,700,300,400italic,700italic,300italic|Roboto+Condensed:400,700....`